webpack.config.js

```diff
@@ -276,11 +276,6 @@ module.exports = (env = process.env.NODE_ENV || 'development') => {
           ],
           use: ['null-loader'],
         },
-        {
-          test: /\.js$/u,
-          include: matchModule('moment/locale'),
-          use: ['null-loader'],
-        },
```

Comment:
I’m guessing this might be why the bundle size increased substantially? (Just adding `date-fns` should not move the needle much)